users/views.py

```diff
@@ -39,3 +40,15 @@ class SearchUserView(generics.ListAPIView):
     filter_backends = (filters.SearchFilter,)
     queryset = User.objects.all()
     serializer_class = UserPublicSerializer
+
+
+class FollowerViewSet(viewsets.ModelViewSet):
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, pk: int):
+        Followers.objects.update_or_create(follower=request.user, following_id=pk)
+        return Response(status=status.HTTP_201_CREATED)
+
+    def destroy(self, request, pk: int) -> Response:
+        Followers.objects.filter(follower=request.user, following_id=pk).delete()
+        return Response(status=status.HTTP_200_OK)
```

Comment:
200 is ok here. An alternative for delete endpoint is `204 No Content`

Reply:
Done